Makefile

```
@@ -318,9 +318,18 @@ config_updater_vault_secret:
 		--context=$(cluster) \
 		--confirm \
 		--kubeconfig=/_kubeconfig
-	mkdir -p $(build_farm_credentials_folder)
-	oc --context "$(cluster)" sa create-kubeconfig -n ci config-updater > "$(build_farm_credentials_folder)/sa.config-updater.$(cluster).config"
-	make dry_run=false ci-secret-generator
+
+	./clusters/psi/create_kubeconfig.sh "$(build_farm_credentials_folder)/sa.config-updater.${cluster}.config" ${cluster} $@ ci ${API_SERVER_URL} config-updater-token-version-$(token_version)
```

```suggestion
	./clusters/psi/create_kubeconfig.sh "$(build_farm_credentials_folder)/sa.config-updater.${cluster}.config" ${cluster} config-updater ci ${API_SERVER_URL} config-updater-token-version-$(token_version)
```

We should correct in the following line too: https://github.com/openshift/release/blob/3b8be0e55dee0382b765364af1655cdcb12d0a64/Makefile#L405